### DISCUSSION TITLE
Night Mode without Inverting Image Colors?

### DISCUSSION THREAD

**Discussion by WandersFar** [sumatrapdfreader/sumatrapdf#3001] [Documentation]
Hey there,

Is it possible to run Sumatra in “night mode” (black background, white text) WITHOUT affecting how images are displayed?

```
FixedPageUI [
	TextColor = #ffffff
	BackgroundColor = #000000
	SelectionColor = #e7e7e7
	WindowMargin = 0 0 0 0
	PageSpacing = 0 0
	GradientColors = #000000
	HideScrollbars = true
]
```

These settings produce white text on a black document background in a black window frame with hidden scrollbars, which is what I want.

However all images are distorted, the colors are inverted as well. I don’t want this. I just want images to be displayed normally. Is that possible?

Thanks for any advice.

**Comment by GitHubRulesOK** [comment]
No, there are no modes that i function is for inversion e.g turn negatives to positive or blue prints to blue lines, for nightime do the same as other apps reduce the brightness by reducing the white tone to grey or pink.
